client/components/spotlight/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+
+const SpotlightContainer = styled.div`
+	background-color: var( --studio-gray-0 );
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 20px;
```

Comment:
```suggestion
	padding: 30px;
	margin-bottom: 10px;
```
To match existing plugin cards. At some point we should work towards using spacing values like `var(--spacing-2)`